style.css

```diff
@@ -127,3 +127,668 @@ li {
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 60px;
+}
+
+.html {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 65px;
+}
+
+.css {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 74px;
+}
+
+#fw {
+  width: 164px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.fw {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  flex-direction: column;
+  margin-top: 230px;
+  margin-left: 100px;
+}
+
+.skills {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  flex-direction: column;
+  margin-top: 450px;
+  margin-left: 100px;
+}
+
+#skills {
+  width: 164px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 80px;
+}
+
+#star {
+  background: #ff6b00;
+  width: 1px;
+  height: 1px;
+  transform: rotate(-45deg);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: -250px;
+  position: absolute;
+  margin-top: 150px;
+}
+
+#box {
+  background: #ff6b00;
+  width: 1px;
+  height: 1px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: -250px;
+  position: absolute;
+  margin-top: 20px;
+}
+
+#circle {
+  background: #ff6b00;
+  width: 1px;
+  height: 1px;
+  border-radius: 150px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: -250px;
+  position: absolute;
+  margin-top: 25px;
+}
```

Comment:
- [ ] Right now, these elements are too big when compared to the original design. You should consider resizing them to match the original design better.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143685026-bc5f76ae-ab57-4c75-a41a-a83203938dd2.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/143685038-0624701e-4ea3-43c2-9378-c70821707017.png)